src/browser/ui/dom/components/__tests__/ReactDOMTextarea-test.js

```diff
@@ -127,6 +127,17 @@ describe('ReactDOMTextarea', function() {
     expect(node.value).toEqual('false');
   });
 
+  it('should allow setting `value` to `null`', function() {
+    var stub = <textarea value="giraffe" onChange={emptyFunction} />;
+    stub = renderTextarea(stub);
+    var node = stub.getDOMNode();
+
+    expect(node.value).toBe('giraffe');
+
+    stub.replaceProps({value: null, onChange: emptyFunction});
+    expect(node.value).toEqual('');
+  });
+
```

Comment:
We need some more tests here. What's the expected behavior when there is a `defaultValue`? What if `defaultValue` also changes? Should `null` and `undefined` behave differently?

I commented in the issue for a bit more discussion before we make a final decision.